qiita_pet/test/test_download.py

```diff
@@ -317,7 +320,10 @@ def test_download(self):
                "1.SKD8.640184\tERX0000001\n1.SKM5.640177\tERX0000005\n"
                "1.SKM7.640188\tERX0000010\n1.SKD7.640191\tERX0000021")
         self.assertEqual(response.code, 200)
-        self.assertRegexpMatches(response.body, exp)
+        # testing as lists so we ignore order
```

Comment:
Would it be better to sort the lists so they could be compared for equality? There's a test earlier where all of the strings are the same, but this test data appears to have multiple IDs.

Reply:
Possible to change but not sure what we will gain from this ...

Reply:
Just that before, our test was 'self.assertRegexpMatches(response.body, exp)', and now we're just asserting that the number of elements in obs and exp is equal: self.assertCountEqual(obs, exp).

Reply:
As Antonio pointed out, assertCountEqual() does in fact test the elements of both lists, regardless of order: 
https://www.google.com/url?q=https://docs.python.org/3/library/unittest.html%23unittest.TestCase.assertCountEqual&sa=D&source=hangouts&ust=1550959180059000&usg=AFQjCNHmkIF8ZDeLHZRdII_u_Cwd0ffBHQ

Hence, this is not an issue.